source/frontend/src/features/leases/models.ts

```diff
@@ -214,22 +215,17 @@ export class FormLeaseProperty {
   }
 
   public static toApi(formLeaseProperty: FormLeaseProperty): ApiGen_Concepts_PropertyLease {
-    const landArea = stringToUndefined(formLeaseProperty.landArea);
-    const numberLeaseArea: number | undefined = isValidString(landArea)
-      ? Number(landArea)
-      : undefined;
     return {
       id: formLeaseProperty.id ?? 0,
       fileId: formLeaseProperty.leaseId ?? 0,
       file: null,
       property: formLeaseProperty.property?.toApi() ?? null,
       propertyId: formLeaseProperty.property?.id ?? 0,
       propertyName: formLeaseProperty.name ?? null,
-      leaseArea: numberLeaseArea ?? null,
-      areaUnitType:
-        numberLeaseArea !== undefined
-          ? toTypeCodeNullable(formLeaseProperty.areaUnitTypeCode) ?? null
-          : null,
+      leaseArea: isNumber(formLeaseProperty.landArea) ? formLeaseProperty.landArea : 0,
```

Comment:
should this be null when 'landArea' is not a number?

Reply:
it is typed as a number so I gave the default value of 0